### ISSUE TITLE
Database structure

### ISSUE THREAD

**Issue by VincentDeMonaweb** [cleverfox/volleyball_scoreboard#1]
Hello,
I would like to use your project, but I couldn't find the database structure.
Firebase, is that it?
Is it possible to get it?
Thank you in advance for your answer.

**Comment by cleverfox** [comment]
Yes, it's use firebase and there is no structure inside. It creates object for event just to distinguish events

**Comment by VincentDeMonaweb** [comment]
I have never used Firebase I will try to understand.
I would like to modify the script to include an image (logo) in front of the name, instead of a color.

Is your script free to modify and use?

**Comment by cleverfox** [comment]
@SNMVGlobaltech Sure, it's opensource, so you can use it and modify. Pull requests are welcome.

About your idea with picture, I see two ways to do it. First one is store in firebase URL for image hosted somewhere, so scoreboard load this image directly from internet. Another solution is store binary or base64 encoded image in database. 
I think you should try to store base64 image in database firstly.
